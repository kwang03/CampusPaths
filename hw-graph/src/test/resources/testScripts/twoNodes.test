#Tests how a graph with one node behaves

#Creates graph
CreateGraph myGraph

#Adds node
AddNode myGraph n1
AddNode myGraph n2

#Lists children
ListChildren myGraph n1
ListChildren myGraph n2

#Lists nodes
ListNodes myGraph