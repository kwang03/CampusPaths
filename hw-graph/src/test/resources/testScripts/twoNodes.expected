#Tests how a graph with one node behaves

#Creates graph
created graph myGraph

#Adds node
added node n1 to myGraph
added node n2 to myGraph

#Lists children
the children of n1 in myGraph are:
the children of n2 in myGraph are:

#Lists nodes
myGraph contains: n1 n2